### PR TITLE
create-a-new-release-in-jira 1.0.0

### PR DESCRIPTION
**Pull request by Emalober**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] __I will not move an already shared step version's tag to another commit__
- [ ] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/create-a-new-release-in-jira/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
```
Comment (koral--) [inline steps/create-a-new-release-in-jira/1.0.0/step.yml:27]: You are using curl, but neither git nor wget.

File: steps/create-a-new-release-in-jira/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    is_required: true
+    summary: Version name
+    title: Version  name
+  version_name: null
+- opts:
+    description: |
+      If the version was published.
+    is_required: true
+    summary: If the version was published
+    title: If the version was published?
```
Comment (koral--) [inline steps/create-a-new-release-in-jira/1.0.0/step.yml:96]: Description of this input does not seem to match the logic behind it.
Past tense here suggests that was already released. Such release event seems to be unrelated to the action performed by the step.
However, in the script you are assigning today's date. So it suggests that the release happens at the time of step execution, not in the past.

You should correct either of that places.

File: steps/create-a-new-release-in-jira/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    is_required: true
+    summary: Jira company domain
+    title: Jira domain
+- jira_user: null
+  opts:
+    description: |
+      Jira user used to authenticate with Jira API
+
+      e.g. "someone@example.pl"
+    is_sensitive: true
```
Comment (koral--) [inline steps/create-a-new-release-in-jira/1.0.0/step.yml:59]: I don't think that username should be sensitive.
What is a reason behind that?

Comment (Emalober) [reply steps/create-a-new-release-in-jira/1.0.0/step.yml:59]: I saw that in many cases, it is used as a sensitive field.

File: steps/create-a-new-release-in-jira/1.0.0/step.yml (hunk 1 above)
Comment (koral--) [reply steps/create-a-new-release-in-jira/1.0.0/step.yml:27]: yes, it should be added.
The only exceptions are bash and go.
